python/tvm/topi/cuda/conv2d_nhwc.py

```diff
@@ -85,8 +87,10 @@ def schedule_conv2d_nhwc_direct(cfg, s, Conv):
     thread_yz = te.thread_axis((0, vthread_n), "vthread", name="vy")
 
     # Schedule for output
-    ni, hi, wi, fi = s[output].op.axis
-    bz = s[output].fuse(hi, wi)
+    ni, _, wi, fi = s[output].op.axis
+    bz = wi
+    fi, vec = s[output].split(fi, factor=vec_factor)
+    s[output].vectorize(vec)
```

Comment:
Does this actually work? ...... In my experience it's better to put this vectorize at the back of reorder.
And in L96, you didn't add the new vector axis to the reorder list?

Reply:
Yes, it works. I added vectorize to the back of reorder list. Sorry, I'm new in writing schedules, so, maybe I did something wrong :) I checked the performance and accuracy on an OpenCL device, and it works fine and fast.